pkgs/servers/search/zincsearch/default.nix

```diff
@@ -57,6 +58,5 @@ buildGoModule rec {
     maintainers = with maintainers; [ dit7ya ];
     # Doesn't build with Go version later v1.21 (which is EOL).
     # Upstream issue: https://github.com/zincsearch/zincsearch/issues/975
```

Comment:
Seems like the comment is no longer necessary or am I missing something, @katexochen?
```suggestion
```